SpreadCheetah/Validations/DataValidation.cs

```diff
@@ -74,6 +74,64 @@ public ValidationErrorType ErrorType
 
     private ValidationErrorType _errorType;
 
+    #region DateOnly validations
+    private static DataValidation _DateOnly(ValidationOperator op, DateOnly min, DateOnly max) => max < min
+        ? throw new ArgumentException(MinGreaterThanMaxMessage, nameof(min))
+        : new DataValidation(ValidationType.DateOnly, op, min.ToStringInvariant(), max.ToStringInvariant());
+
+    private static DataValidation _DateOnly(ValidationOperator op, DateOnly value) => new(ValidationType.DateOnly, op, value.ToStringInvariant());
+
+    /// <summary>Validate that decimals are between <paramref name="min"/> and <paramref name="max"/>.</summary>
+    public static DataValidation DateOnlyBetween(DateOnly min, DateOnly max) => _DateOnly(ValidationOperator.Between, min, max);
+    /// <summary>Validate that decimals are not between <paramref name="min"/> and <paramref name="max"/>.</summary>
+    public static DataValidation DateOnlyNotBetween(DateOnly min, DateOnly max) => _DateOnly(ValidationOperator.NotBetween, min, max);
+    /// <summary>Validate that decimals are equal to <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyEqualTo(DateOnly value) => _DateOnly(ValidationOperator.EqualTo, value);
+    /// <summary>Validate that decimals are not equal to <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyNotEqualTo(DateOnly value) => _DateOnly(ValidationOperator.NotEqualTo, value);
+    /// <summary>Validate that decimals are greater than <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyGreaterThan(DateOnly value) => _DateOnly(ValidationOperator.GreaterThan, value);
+    /// <summary>Validate that decimals are greater than or equal to <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyGreaterThanOrEqualTo(DateOnly value) => _DateOnly(ValidationOperator.GreaterThanOrEqualTo, value);
+    /// <summary>Validate that decimals are less than <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyLessThan(DateOnly value) => _DateOnly(ValidationOperator.LessThan, value);
+    /// <summary>Validate that decimals are less than or equal to <paramref name="value"/>.</summary>
+    public static DataValidation DateOnlyLessThanOrEqualTo(DateOnly value) => _DateOnly(ValidationOperator.LessThanOrEqualTo, value);
+
+    #endregion DateOnly validations
+
+    #region DateTime validations
+    private static DataValidation _DateTime(ValidationOperator op, DateTime min, DateTime max)
+    {
+        if (max < min)
+            throw new ArgumentException(MinGreaterThanMaxMessage, nameof(min));
+        else
+            return new DataValidation(ValidationType.DateTime, op, min.ToStringInvariant(), max.ToStringInvariant());
```

Comment:
Are you sure about storing `DateTime` values as strings like this is correct for the Excel file?

Reply:
Regarding datetime being represented as string, Excel follows the Culture in which it is installed. Therefore, for north american people, the datetime string representation would be like the following: "yyyy/mm/dd HH:MM:ss:ms"; for latin people, the same datetime would be represented in the following format: "dd/mm/yyyy HH:MM:ss:ms".

Regardless of how datetime is string represented, within Excel, a datetime is basically an integer (i.e., ticks since EPOCH). Therefore, one could also define datetime as an integer number.

Reply:
Dear Sveinungf, after some digging, I found the standard definition of date for Excel. See reference [here](https://support.microsoft.com/en-us/office/date-systems-in-excel-e7fe7167-48a9-4b96-bb53-5612a800b487). 

In this last commit, the Datetime representation has been adjusted to the provided reference format.

Sincerely,